components/nimbus/ios/Nimbus/FeatureVariables.swift

```diff
@@ -3,7 +3,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import Foundation
-import UIKit
+//import UIKit
```

Comment:
Meh I'll have to think about this harder, I want to use the Variables API in my test scripts, but those run on my Mac... `UIKit` is only for iOS. Either we mock it, or we replace `UIImage` with a protocol and have tests mock that protocol (while real code uses `UIImage`)... Not sure yet, I'll get back to this at a later point when I'm trying to clean things up